apps/cowswap-frontend/src/legacy/state/orders/utils.ts

```diff
@@ -135,7 +135,7 @@ export function classifyOrder(
  * @param executionPrice
  */
 export function isOrderUnfillable(
-  order: Order,
+  order: Order | ParsedOrder,
```

Comment:
Use the aggregated type here

Reply:
Replaced in the whole app and renamed it to `GenericOrder`